test/primer/classify_test.rb

```diff
@@ -282,11 +282,35 @@ def test_does_not_raise_error_when_passing_in_a_primer_css_class_otherwise
     assert_generated_class("bg-blue text-center float-left ml-1 ",  { classes: "bg-blue text-center float-left ml-1" })
   end
 
+  def test_limits_allocations
+    # Warm up allocations
+    values = { align_self: :center, width: :fit, p: 4, m: 1, border: :top, box_shadow: true, color: :red, visibility: :hidden}
```

Comment:
are we using **all** available keys here?

Reply:
I'm not, but that's not a bad idea.

I don't want to spend a ton more time on this specific PR, but that could be a good follow-up.

Reply:
my idea here is that if we test all keys, we could catch regressions easier in the future :)

Reply:
You've convinced me, I was thinking of keeping it as-is so we could keep track of which aren't optimized and which are, but tracking regressions is more important. I'll add a bunch more keys to this call and bump the allocation values.